esp32-firmware/src/main.cpp

```
@@ -3,20 +3,29 @@
 #include <PubSubClient.h>
 #include <ArduinoJson.h>
 #include <arduinoFFT.h>
+#include <driver/adc.h>
+#include <hal/adc_ll.h>
 #include "config.h"
 
 // --- PARAMETRY POMIAROWE ---
-#define SAMPLES 512
-#define SAMPLING_FREQ 3000 
+#define SAMPLES 1024
+#define SAMPLING_FREQ 10000
```

Clang (14.0.6)

[warning] macro 'SAMPLING_FREQ' used to declare a constant; consider using a 'constexpr' constant

(cppcoreguidelines-macro-usage)